### DISCUSSION TITLE
Cannot create new pages or navigate to "root" pages like 'contact'

### DISCUSSION THREAD

**Discussion by joel-marks** [thuliteio/doks#1065] [Q&A]
Hi there, sorry for the rookie question. I'm running the server on a mac hacking at the code to make a site.

Adding a new page using the npm command will create the page, but I cannot navigate to it. 404.
Having a go at navigating to pages that appear to already be there, such as /contact/ also gives a 404.

Any help appreciated, sorry if I'm didn't rtfm enough.

**Comment by h-enk** [comment]
No worries! Did you set `draft = false` in the frontmatter of the pages?

**Comment by joel-marks** [comment]
Yep. That was it. 🤦 

Thank you *so* much. rtfm indeed 😄
